clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -132,6 +132,119 @@ goodbye-world"
     );
 }
 
+#[test]
+fn suggest_argument_value() {
+    let mut cmd = Command::new("dynamic")
+    .arg(
+        clap::Arg::new("input")
+            .long("input")
+            .short('i')
+            .value_hint(clap::ValueHint::FilePath),
+    )
+    .arg(
+        clap::Arg::new("format")
+            .long("format")
+            .short('F')
+            .value_parser(["json", "yaml", "toml"]),
+    )
+    .args_conflicts_with_subcommands(true);
+    
+    {
+        use std::fs::File;
+        use std::path::Path;
+
+        let testdir = snapbox::dir::DirRoot::mutable_temp().unwrap();
+        let testdir_path = testdir.path().unwrap();
+
+        File::create(Path::new(testdir_path).join("a_file")).unwrap();
```

Comment:
Please instead use `std::fs::write

Reply:
I have modified the code, but I am not sure if this is what you meant.